src/main/java/org/jenkinsci/plugins/gitclient/CheckoutCommand.java

```diff
@@ -49,4 +49,12 @@
      * @return a {@link org.jenkinsci.plugins.gitclient.CheckoutCommand} object.
      */
     CheckoutCommand timeout(Integer timeout);
+    
```

Comment:
Could you remove the trailing white space on this line?

The formatting in the project is already somewhat messy, and I'd much rather not add another line of surprising formatting